genai-perf/genai_perf/llm_inputs/llm_inputs.py

```diff
@@ -637,37 +645,18 @@ def _get_prompts_from_input_file(
         with open(input_filename, mode="r", newline=None) as file:
             for line in file:
                 if line.strip():
-                    prompts.append(load_json_str(line).get("text_input", "").strip())
-                    images.append(load_json_str(line).get("image", "").strip())
+                    # None if not provided
+                    prompt = load_json_str(line).get("text_input")
+                    image = load_json_str(line).get("image")
+                    prompts.append(prompt.strip() if prompt else prompt)
+                    images.append(image.strip() if image else image)
         return prompts, images
 
     @classmethod
     def verify_file(cls, input_filename: Path) -> None:
         if not input_filename.exists():
             raise FileNotFoundError(f"The file '{input_filename}' does not exist.")
 
-    @classmethod
-    def _convert_to_openai_multi_modal_content(
```

Comment:
Moved as a new method `_extract_chat_contents` that's used when converting from generic json to chat format. Since this is endpoint specific logic, this doesn't need to be here.